src/components/JobPostForm/index.tsx

```diff
@@ -26,7 +31,13 @@ const JobPostForm = ({ header }: HeaderProps) => {
     day: 31,
   }
 
+  const user = useSelector((state: AppState) => state.user)
+  const { loading, error } = user
+
+  const dispatch = useDispatch()
+
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const startingDate = startingAt
```

Comment:
Why need to assign again. You can use it directly

Reply:
Okey.